tsl/test/isolation/expected/cagg_concurrent_process_wal.out

```diff
@@ -0,0 +1,27 @@
+Parsed test spec with 3 sessions
+
+starting permutation: lock_enable s1_refresh s2_refresh lock_show lock_release
+step lock_enable: SELECT debug_waitpoint_enable('multi_invalidation_process_invalidations');
+debug_waitpoint_enable
+----------------------
+                      
+(1 row)
+
+step s1_refresh: CALL _timescaledb_functions.process_hypertable_invalidations(ARRAY['temperature']); <waiting ...>
+step s2_refresh: CALL _timescaledb_functions.process_hypertable_invalidations(ARRAY['temperature']); <waiting ...>
+step lock_show: SELECT locktype, granted, objtyp, objid FROM my_locks WHERE locktype in ('object', 'advisory');
+locktype|granted|objtyp  |     objid
+--------+-------+--------+----------
+advisory|f      |-       |1465306080
+object  |t      |pg_class|     16678
+object  |f      |pg_class|     16678
```

Comment:
One improvement could be to print a session/application name and print as a column. (E.g., using `set application_name='s1'`)